akka-http-core/src/test/scala/akka/http/scaladsl/GracefulTerminationSpec.scala

```diff
@@ -34,6 +32,7 @@ import scala.util.Success
 
 class GracefulTerminationSpec
   extends AkkaSpecWithMaterializer("""
+    akka.loglevel = debug
```

Comment:
already set in `AkkaSpecWithMaterializer`

Reply:
oops